packages/fluentui/docs/src/examples/components/Menu/Visual/MenuExampleCompose.shorthand.tsx

```diff
@@ -122,8 +124,10 @@ const themeOverrides = {
     MenuitemIndicatorSaturated: {
       root: {
         filter: 'saturate(8)',
+        backgroundSize: '20px',
         height: '20px',
         width: '20px',
+        float: 'right' as FloatProperty,
```

Comment:
@mnajdova @layershifter I am here only making it was before the changes in the menuItemStyles so we avoid screener regressions.